src/libstd/sys/unix/ext/net.rs

```diff
@@ -692,6 +1104,15 @@ impl UnixDatagram {
     /// Sends data on the socket to the specified address.
     ///
     /// On success, returns the number of bytes written.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::os::unix::net::UnixDatagram;
+    ///
+    /// let sock = UnixDatagram::unbound().unwrap();
+    /// sock.send_to(b"omelette AU fromage", "/some/sock").expect("send_to function failed");
```

Comment:
For me, a person who doesn't know any French, it doesn't help me learn French, it just makes me confused :)